rotkehlchen/api/v1/resources.py

```diff
@@ -2953,3 +2954,19 @@ class DefiMetadataResource(BaseMethodView):
 
     def get(self) -> Response:
         return self.rest_api.get_defi_metadata()
+
+
+class ExportHistoryEventResource(BaseMethodView):
+
+    post_schema = ExportHistoryEventSchema()
+    patch_schema = ExportHistoryEventSchema(exclude=('directory_path',))
```

Comment:
Just use post/post. The reason I used patch in my case was I had no other verb left. Patch should mostly be used for editing.

```suggestion
    put_schema = ExportHistoryEventSchema(exclude=('directory_path',))
```